### PR TITLE
Remplace pills made by the condimaster with bouillon cubes.

### PR DESCRIPTION
**Pull request by lbnesquik**
Just a couple lines remplace so the condimaster can create pills which look like bouillon cubes.
The .DMI change is just me adding the monkey cube sprite with a different name. It was fair easier than to have the template look into another file for the icon.

example

example2

### CODE REVIEW

**Comment by lbnesquik** [comment]
https://github.com/PolarisSS13/Polaris/pull/5689

**Comment by Neerti** [comment]
You appeared to have linked to this exact PR (#5689), which I don't think you intended to do.

**Comment by lbnesquik** [comment]
Not really, no!

File: code/modules/reagents/Chemistry-Machinery.dm [hunk 1]
```diff
@@ -231,9 +231,14 @@
 			var/amount_per_pill = reagents.total_volume/count
 			if (amount_per_pill > 60) amount_per_pill = 60
 
-			var/name = sanitizeSafe(input(usr,"Name:","Name your pill!","[reagents.get_master_reagent_name()] ([amount_per_pill] units)") as null|text, MAX_NAME_LEN)
-			if(!name) //Blank name (sanitized to nothing, or left empty) or cancel
-				return
+			if(condi)//For the condimaster
+				var/name = sanitizeSafe(input(usr,"Name:","Name your cube!","[reagents.get_master_reagent_name()] ([amount_per_pill] units)") as null|text, MAX_NAME_LEN)
+				if(!name) //Repeated because i suck
```
Comment (Atermonera) [inline code/modules/reagents/Chemistry-Machinery.dm]: You can pull both of these `if(!name)`s out of the if else block and put it after the else. The code needs to run regardless of the outcome of the if, so it shouldn't go in the if

Comment (lbnesquik) [reply code/modules/reagents/Chemistry-Machinery.dm]: If there is no name, a return occurs and no pill is produced.

File: code/modules/reagents/Chemistry-Machinery.dm [hunk 2]
```diff
@@ -231,10 +231,15 @@
 			var/amount_per_pill = reagents.total_volume/count
 			if (amount_per_pill > 60) amount_per_pill = 60
 
-			var/name = sanitizeSafe(input(usr,"Name:","Name your pill!","[reagents.get_master_reagent_name()] ([amount_per_pill] units)") as null|text, MAX_NAME_LEN)
+			if(condi)//For the condimaster
+				var/name = sanitizeSafe(input(usr,"Name:","Name your cube!","[reagents.get_master_reagent_name()] ([amount_per_pill] units)") as null|text, MAX_NAME_LEN)
```
Comment (Atermonera) [inline code/modules/reagents/Chemistry-Machinery.dm]: This only exists within the scope of this if statement. The variable declaration needs to be moved outside of it

**Pull request merged**